### ISSUE TITLE
_results: exception AttributeError: type object 'RaceResult' has no attribute 'age'

### ISSUE THREAD

**Issue by louking** [louking/rrwebapp#230]
Not clear how this happened, as 'age' shouldn't be searchable

Saw following exception

```
    Message type:       ERROR
    Location:           /var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py:1423
    Module:             app
    Function:           log_exception
    Time:               2016-07-26 22:23:09,230

    Message:

    Exception on /_results [GET]

    Traceback (most recent call last):
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
        response = self.full_dispatch_request()
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
        rv = self.handle_user_exception(e)
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
        reraise(exc_type, exc_value, tb)
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
        rv = self.dispatch_request()
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
        return self.view_functions[rule.endpoint](**req.view_args)
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/views.py", line 84, in view
        return self.dispatch_request(*args, **kwargs)
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/views.py", line 149, in dispatch_request
        return meth(*args, **kwargs)
      File "/var/www/scoretility.com/rrwebapp/rrwebapp/request.py", line 309, in wrapped_function
        resp = make_response(f(*args, **kwargs))
      File "/var/www/scoretility.com/rrwebapp/rrwebapp/results.py", line 1400, in get
        rowTable = DataTables(request.args, RaceResult, RaceResult.query.filter_by(**resultfilter).join("runner").join("series").filter_by(**seriesfilter).join("race"), columns,         dialect='mysql')
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/sqlalchemy_datatables-0.4.1lk-py2.7.egg/datatables/__init__.py", line 202, in __init__
        self.run()
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/sqlalchemy_datatables-0.4.1lk-py2.7.egg/datatables/__init__.py", line 256, in run
        self.sorting()
      File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/sqlalchemy_datatables-0.4.1lk-py2.7.egg/datatables/__init__.py", line 439, in sorting
        obj = getattr(self.sqla_object, tmp_name)
    AttributeError: type object 'RaceResult' has no attribute 'age'
```

with following request

```
    <IP obscured> - - [26/Jul/2016:22:23:09 -0400] "GET /_results?series=equalizer&participant=185&draw=2&columns%5B0%5D%5Bdata%5D=name&columns%5B0%5D%5Bname%5D=name&columns%5B0%5D%5Bsearchable%5D=true&columns%5B0%5D%5Borderable%5D=true&columns%5B0%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B0%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B1%5D%5Bdata%5D=series&columns%5B1%5D%5Bname%5D=series&columns%5B1%5D%5Bsearchable%5D=true&columns%5B1%5D%5Borderable%5D=true&columns%5B1%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B1%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B2%5D%5Bdata%5D=date&columns%5B2%5D%5Bname%5D=date&columns%5B2%5D%5Bsearchable%5D=true&columns%5B2%5D%5Borderable%5D=true&columns%5B2%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B2%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B3%5D%5Bdata%5D=race&columns%5B3%5D%5Bname%5D=race&columns%5B3%5D%5Bsearchable%5D=true&columns%5B3%5D%5Borderable%5D=true&columns%5B3%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B3%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B4%5D%5Bdata%5D=miles&columns%5B4%5D%5Bname%5D=miles&columns%5B4%5D%5Bsearchable%5D=true&columns%5B4%5D%5Borderable%5D=true&columns%5B4%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B4%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B5%5D%5Bdata%5D=gender&columns%5B5%5D%5Bname%5D=gender&columns%5B5%5D%5Bsearchable%5D=true&columns%5B5%5D%5Borderable%5D=true&columns%5B5%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B5%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B6%5D%5Bdata%5D=age&columns%5B6%5D%5Bname%5D=age&columns%5B6%5D%5Bsearchable%5D=true&columns%5B6%5D%5Borderable%5D=true&columns%5B6%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B6%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B7%5D%5Bdata%5D=genderplace&columns%5B7%5D%5Bname%5D=genderplace&columns%5B7%5D%5Bsearchable%5D=true&columns%5B7%5D%5Borderable%5D=true&columns%5B7%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B7%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B8%5D%5Bdata%5D=division&columns%5B8%5D%5Bname%5D=division&columns%5B8%5D%5Bsearchable%5D=true&columns%5B8%5D%5Borderable%5D=true&columns%5B8%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B8%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B9%5D%5Bdata%5D=divisionplace&columns%5B9%5D%5Bname%5D=divisionplace&columns%5B9%5D%5Bsearchable%5D=true&columns%5B9%5D%5Borderable%5D=true&columns%5B9%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B9%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B10%5D%5Bdata%5D=time&columns%5B10%5D%5Bname%5D=time&columns%5B10%5D%5Bsearchable%5D=true&columns%5B10%5D%5Borderable%5D=true&columns%5B10%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B10%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B11%5D%5Bdata%5D=pace&columns%5B11%5D%5Bname%5D=pace&columns%5B11%5D%5Bsearchable%5D=true&columns%5B11%5D%5Borderable%5D=true&columns%5B11%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B11%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B12%5D%5Bdata%5D=agtime&columns%5B12%5D%5Bname%5D=agtime&columns%5B12%5D%5Bsearchable%5D=true&columns%5B12%5D%5Borderable%5D=true&columns%5B12%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B12%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B13%5D%5Bdata%5D=agpercent&columns%5B13%5D%5Bname%5D=agpercent&columns%5B13%5D%5Bsearchable%5D=true&columns%5B13%5D%5Borderable%5D=true&columns%5B13%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B13%5D%5Bsearch%5D%5Bregex%5D=false&order%5B0%5D%5Bcolumn%5D=6&order%5B0%5D%5Bdir%5D=asc&start=0&length=10&search%5Bvalue%5D=&search%5Bregex%5D=false&_=1469586162458 HTTP/1.1" 500 291 "http://www.scoretility.com/results?series=equalizer&participant=185" "Mozilla/5.0 (Mobile; Windows Phone 8.1; Android 4.0; ARM; Trident/7.0; Touch; rv:11.0; IEMobile/11.0; Microsoft; Lumia 640 LTE) like iPhone OS 7_0_3 Mac OS X AppleWebKit/537 (KHTML, like Gecko) Mobile Safari/537"
```

**Comment by louking** [comment]
```
        Message type:       ERROR
        Location:           /var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py:1587
        Module:             app
        Function:           log_exception
        Time:               2018-06-25 11:22:45,843



        Message:

        Exception on /_results [GET]

Traceback (most recent call last):
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/views.py", line 84, in view
    return self.dispatch_request(*args, **kwargs)
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/flask/views.py", line 149, in dispatch_request
    return meth(*args, **kwargs)
  File "/var/www/scoretility.com/rrwebapp/rrwebapp/request.py", line 321, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "/var/www/scoretility.com/rrwebapp/rrwebapp/results.py", line 1503, in get
    rowTable = DataTables(request.args, RaceResult, RaceResult.query.filter_by(**resultfilter).join("runner").join("series").filter_by(**seriesfilter).join("race"), columns, dialect='mysql')
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/sqlalchemy_datatables-0.4.1lk3-py2.7.egg/datatables/__init__.py", line 244, in __init__
    self.run()
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/sqlalchemy_datatables-0.4.1lk3-py2.7.egg/datatables/__init__.py", line 298, in run
    self.sorting()
  File "/var/www/scoretility.com/rrwebapp/lib/python2.7/site-packages/sqlalchemy_datatables-0.4.1lk3-py2.7.egg/datatables/__init__.py", line 492, in sorting
    obj = getattr(self.sqla_object, tmp_name)
AttributeError: type object 'RaceResult' has no attribute 'age'
```

`<ip> - - [25/Jun/2018:11:22:45 -0400] "GET /_results?club=fsrc&draw=10&columns%5B0%5D%5Bdata%5D=name&columns%5B0%5D%5Bname%5D=name&columns%5B0%5D%5Bsearchable%5D=true&columns%5B0%5D%5Borderable%5D=true&columns%5B0%5D%5Bsearch%5D%5Bvalue%5D=William+Shutt&columns%5B0%5D%5Bsearch%5D%5Bregex%5D=true&columns%5B1%5D%5Bdata%5D=series&columns%5B1%5D%5Bname%5D=series&columns%5B1%5D%5Bsearchable%5D=true&columns%5B1%5D%5Borderable%5D=true&columns%5B1%5D%5Bsearch%5D%5Bvalue%5D=decathlon&columns%5B1%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B2%5D%5Bdata%5D=date&columns%5B2%5D%5Bname%5D=date&columns%5B2%5D%5Bsearchable%5D=true&columns%5B2%5D%5Borderable%5D=true&columns%5B2%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B2%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B3%5D%5Bdata%5D=race&columns%5B3%5D%5Bname%5D=race&columns%5B3%5D%5Bsearchable%5D=true&columns%5B3%5D%5Borderable%5D=true&columns%5B3%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B3%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B4%5D%5Bdata%5D=miles&columns%5B4%5D%5Bname%5D=miles&columns%5B4%5D%5Bsearchable%5D=true&columns%5B4%5D%5Borderable%5D=true&columns%5B4%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B4%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B5%5D%5Bdata%5D=gender&columns%5B5%5D%5Bname%5D=gender&columns%5B5%5D%5Bsearchable%5D=true&columns%5B5%5D%5Borderable%5D=true&columns%5B5%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B5%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B6%5D%5Bdata%5D=age&columns%5B6%5D%5Bname%5D=age&columns%5B6%5D%5Bsearchable%5D=true&columns%5B6%5D%5Borderable%5D=true&columns%5B6%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B6%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B7%5D%5Bdata%5D=genderplace&columns%5B7%5D%5Bname%5D=genderplace&columns%5B7%5D%5Bsearchable%5D=true&columns%5B7%5D%5Borderable%5D=true&columns%5B7%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B7%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B8%5D%5Bdata%5D=division&columns%5B8%5D%5Bname%5D=division&columns%5B8%5D%5Bsearchable%5D=true&columns%5B8%5D%5Borderable%5D=true&columns%5B8%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B8%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B9%5D%5Bdata%5D=divisionplace&columns%5B9%5D%5Bname%5D=divisionplace&columns%5B9%5D%5Bsearchable%5D=true&columns%5B9%5D%5Borderable%5D=true&columns%5B9%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B9%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B10%5D%5Bdata%5D=time&columns%5B10%5D%5Bname%5D=time&columns%5B10%5D%5Bsearchable%5D=true&columns%5B10%5D%5Borderable%5D=true&columns%5B10%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B10%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B11%5D%5Bdata%5D=pace&columns%5B11%5D%5Bname%5D=pace&columns%5B11%5D%5Bsearchable%5D=true&columns%5B11%5D%5Borderable%5D=true&columns%5B11%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B11%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B12%5D%5Bdata%5D=agtime&columns%5B12%5D%5Bname%5D=agtime&columns%5B12%5D%5Bsearchable%5D=true&columns%5B12%5D%5Borderable%5D=true&columns%5B12%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B12%5D%5Bsearch%5D%5Bregex%5D=false&columns%5B13%5D%5Bdata%5D=agpercent&columns%5B13%5D%5Bname%5D=agpercent&columns%5B13%5D%5Bsearchable%5D=true&columns%5B13%5D%5Borderable%5D=true&columns%5B13%5D%5Bsearch%5D%5Bvalue%5D=&columns%5B13%5D%5Bsearch%5D%5Bregex%5D=false&order%5B0%5D%5Bcolumn%5D=6&order%5B0%5D%5Bdir%5D=asc&start=0&length=25&search%5Bvalue%5D=2018&search%5Bregex%5D=false&_=1529940082817 HTTP/1.1" 500 291 "https://scoretility.com/results?club=fsrc" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.87 Safari/537.36"
`

`<ip> - - [25/Jun/2018:11:22:45 -0400] "GET /_results?club=fsrc&draw=10&columns[0][data]=name&columns[0][name]=name&columns[0][searchable]=true&columns[0][orderable]=true&columns[0][search][value]=William+Shutt&columns[0][search][regex]=true&columns[1][data]=series&columns[1][name]=series&columns[1][searchable]=true&columns[1][orderable]=true&columns[1][search][value]=decathlon&columns[1][search][regex]=false&columns[2][data]=date&columns[2][name]=date&columns[2][searchable]=true&columns[2][orderable]=true&columns[2][search][value]=&columns[2][search][regex]=false&columns[3][data]=race&columns[3][name]=race&columns[3][searchable]=true&columns[3][orderable]=true&columns[3][search][value]=&columns[3][search][regex]=false&columns[4][data]=miles&columns[4][name]=miles&columns[4][searchable]=true&columns[4][orderable]=true&columns[4][search][value]=&columns[4][search][regex]=false&columns[5][data]=gender&columns[5][name]=gender&columns[5][searchable]=true&columns[5][orderable]=true&columns[5][search][value]=&columns[5][search][regex]=false&columns[6][data]=age&columns[6][name]=age&columns[6][searchable]=true&columns[6][orderable]=true&columns[6][search][value]=&columns[6][search][regex]=false&columns[7][data]=genderplace&columns[7][name]=genderplace&columns[7][searchable]=true&columns[7][orderable]=true&columns[7][search][value]=&columns[7][search][regex]=false&columns[8][data]=division&columns[8][name]=division&columns[8][searchable]=true&columns[8][orderable]=true&columns[8][search][value]=&columns[8][search][regex]=false&columns[9][data]=divisionplace&columns[9][name]=divisionplace&columns[9][searchable]=true&columns[9][orderable]=true&columns[9][search][value]=&columns[9][search][regex]=false&columns[10][data]=time&columns[10][name]=time&columns[10][searchable]=true&columns[10][orderable]=true&columns[10][search][value]=&columns[10][search][regex]=false&columns[11][data]=pace&columns[11][name]=pace&columns[11][searchable]=true&columns[11][orderable]=true&columns[11][search][value]=&columns[11][search][regex]=false&columns[12][data]=agtime&columns[12][name]=agtime&columns[12][searchable]=true&columns[12][orderable]=true&columns[12][search][value]=&columns[12][search][regex]=false&columns[13][data]=agpercent&columns[13][name]=agpercent&columns[13][searchable]=true&columns[13][orderable]=true&columns[13][search][value]=&columns[13][search][regex]=false&order[0][column]=6&order[0][dir]=asc&start=0&length=25&search[value]=2018&search[regex]=false&_=1529940082817 HTTP/1.1" 500 291 "https://scoretility.com/results?club=fsrc" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.87 Safari/537.36"
`

**Comment by louking** [comment]
most likely fixed with 66bfe44bf6851ad32f01f55416c14718f5cfc939
